src/workspace/views/QuickLaneOverview.tsx

```diff
@@ -215,38 +247,134 @@ const QuickLaneOverview: FunctionComponent<QuickLaneOverviewProps> = ({ user })
 	// Rendering
 
 	const renderCell = (data: QuickLaneUrlObject, id: string) => (
-		<QuickLaneFilterTableCell id={id} data={data} />
+		<QuickLaneFilterTableCell
+			id={id}
+			data={data}
+			actions={(data) => {
+				type actions = 'edit' | 'copy' | 'delete';
+
+				const items = [
+					{
+						icon: 'edit',
+						id: 'edit',
+						label: t('workspace/views/quick-lane-overview___bewerk'),
+					},
+					{
+						icon: 'copy',
+						id: 'copy',
+						label: t('workspace/views/quick-lane-overview___kopieer-link'),
+					},
+					{
+						icon: 'delete',
+						id: 'delete',
+						label: t('workspace/views/quick-lane-overview___verwijder'),
+					},
+				] as (MenuItemInfoSchema & { id: actions })[];
+
+				return (
+					data && (
+						<MoreOptionsDropdown
+							isOpen={data?.id === selected?.id}
+							onOpen={() => setSelected(data)}
+							onClose={() => {
+								const isAModalOpen =
+									isQuickLaneModalOpen || isConfirmationModalOpen;
+
+								!isAModalOpen && setSelected(undefined);
+							}}
+							label={t('workspace/views/quick-lane-overview___meer-acties')}
+							menuItems={items}
+							onOptionClicked={async (action) => {
+								if (selected === undefined) {
+									return;
+								}
+
+								switch (action.toString() as actions) {
+									case 'edit':
+										setSelected(
+											await QuickLaneService.fetchQuickLaneById(selected.id)
+										);
+										setIsQuickLaneModalOpen(true);
```

Comment:
if the collection to which the quicklane points does not exist, then the fetch will fail, but the user does not see any error toast. it just fails silently:
![image](https://user-images.githubusercontent.com/1710840/195539991-06ab7ad8-121d-42bd-a38f-5f2326787128.png)
![image](https://user-images.githubusercontent.com/1710840/195540009-c8867d55-32d7-4ac4-8475-f2a116d70d33.png)

going to the quicklane link shows this screen:
![image](https://user-images.githubusercontent.com/1710840/195540075-0f96a556-f283-4f9b-a6ee-276e1e276b64.png)
I would expect to see: this collection no longer exists, or something similar.